common/djangoapps/track/views/__init__.py

```diff
@@ -76,6 +79,7 @@ def user_track(request):
     context_override['page'] = page
 
     with eventtracker.get_tracker().context('edx.course.browser', context_override):
+        log.info(u"Event emitted '%s' for %s at page: %s ", name, username, page)
```

Comment:
@shnayder @awaisdar001 

We're certainly not experts in the video js code, but we can do some  digging and brainstorm things on our end to log. Sounds like this PR should be closed then while we come up with a better strategy.